skaffold-plugins-core/build.gradle

```diff
@@ -35,18 +35,18 @@ configurations {
 
 dependencies {
     // Make sure these are consistent with skaffold-maven-plugin and skaffold-gradle-plugin.
-    compile 'com.google.guava:guava:23.6.1-jre'
+    compile 'com.google.guava:guava:26.0-jre'
 
     testCompile 'junit:junit:4.12'
-    testCompile 'org.mockito:mockito-core:2.18.3'
+    testCompile 'org.mockito:mockito-core:2.21.0'
 
     // NullAway errorprone plugin
-    annotationProcessor 'com.uber.nullaway:nullaway:0.4.6'
+    annotationProcessor 'com.uber.nullaway:nullaway:0.5.2'
     errorprone 'com.google.errorprone:error_prone_core:2.3.1'
 }
 
 task wrapper(type: Wrapper) {
-    gradleVersion = '4.8'
+    gradleVersion = '4.9'
```

Comment:
I think this needs to be updated to the new style see: `gradle build --warning-mode all`
```
wrapper {
  gradleVersion = '4.9'
}
```